src/libraries/System.Runtime/System.Runtime.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.9.34431.11
```

Comment:
version line should probably be reverted.

Reply:
I'll revert the whole file. Accidentally included.